src/main/webapp/js/userMapData.json

```diff
@@ -71,38 +76,47 @@
       "AP hogeschool Antwerpen",
       "Arteveldehogeschool",
       "Artevelde University College Ghent",
+      "Artevelde University",
       "BA Stassart",
       "Bacaputsteen",
       "Burgerschool",
       "Busleyden Atheneum",
       "EPHEC",
       "Erasmus Student Network",
       "Heilig-Hart Heverlee",
+      "Howest",
       "IMI - Roosdaal",
+      "KA Avelgem",
       "KdG University College",
       "KU Leuven",
       "Sint Pieters College",
       "Sint-Lievens College",
       "Sint-Lodewijk Campus Van Celst",
       "Sint-Willebrord Heilige Familie",
       "Solvay Brussels School of Economics & Management",
+      "Solvay Brussels School",
```

Comment:
If there are multiple entries referring to the same institution, generally we should keep the longer version (`Solvay Brussels School of Economics & Management` in this case)